client/angular/components/text_diff/text_diff.coffee

```diff
@@ -0,0 +1,21 @@
+DiffMatchPatch = require 'diff-match-patch'
+
+angular.module('loomioApp').directive 'textDiff', ->
+  scope: {before: '=', after: '='}
+  restrict: 'E'
+  templateUrl: 'generated/components/text_diff/text_diff.html'
+  controller: ['$scope', ($scope) ->
+    $scope.diff = ->
+      differ = new DiffMatchPatch()
+      diff = differ.diff_main($scope.before||"", $scope.after||"")
+      differ.diff_cleanupSemantic(diff)
```

Comment:
I'd love to pull this out into a shared library so we can use it in several places (since it doesn't have to do with angular)

Something like

```
# shared/helpers/text.coffee
DiffMatchPatch = require 'diff-match-patch'
module.exports = ->
  applyDiff: (before, after) ->
    differ = new DiffMatchPatch()
    ... (other stuff to make the diff great!)
    diff
```

Then the controller code turns simply into
```
{ applyDiff } = require 'shared/helpers/text.coffee'
$scope.diff = ->
  applyDiff($scope.before, $scope.after)
```